light_curves/multiband_lc.ipynb

```diff
@@ -195,13 +197,24 @@
    "outputs": [],
    "source": [
     "start_serial = time.time()\n",
-    "mission_list = ['FERMIGTRIG', 'SAXGRBMGRB']\n",
-    "radius = 0.1*u.degree\n",
     "\n",
-    "#go out and find all light curves in the above curated list which match our target positions\n",
-    "df_lc_heasarc = HEASARC_get_lightcurves(coords_list,labels_list,radius, mission_list)\n",
+    "#what is the size of error_radius for the fermi catalog that we will accept for our cross-matching?\n",
+    "#in degrees; chosen based on histogram of all values for these catalogs\n",
+    "max_fermi_error_radius = str(1.0)  \n",
+    "max_sax_error_radius = str(3.0)\n",
+    "\n",
+    "#list of missions to query and their corresponding error radii\n",
+    "heasarc_cat = [\"FERMIGTRIG\", \"SAXGRBMGRB\"]\n",
+    "error_radius = [max_fermi_error_radius , max_sax_error_radius]\n",
+    "\n",
+    "#first make a VOTable from our master list of coordinates\n",
+    "#we need a VOTable as input to the pyvo TAP query \n",
+    "xmlfilename = make_VOTable(coords_list, labels_list,'sources.xml')\n",
```

Comment:
I wonder if the `HEASARC_get_lightcurves` function should handle this under the hood, so the user doesn't really have to know about it?

Reply:
good point.  done.